.github/workflows/quick-ci.yml

```diff
@@ -48,3 +48,15 @@ jobs:
           name: quick-ci-jvm-err
           path: '**/*_pid*.log'
           if-no-files-found: ignore
+      - name: On failure, try to reformat and propose fixes
+        uses: burrunan/gradle-cache-action@v1
+        if: failure()
+        with:
+          job-id: spotless-formatting
+          arguments: spotlessJavaApply
```

Comment:
Any reason to prefer `spotlessJavaApply` instead of `spotlessApply`?

Reply:
Ah good call, that's better. I'm also adding updateProto to this as a separate step, so the changes are marked separately.